templates/default/rebuild-iptables.erb

```diff
@@ -130,7 +133,8 @@ end
 
 system_files = %w(/etc/debian_version /etc/redhat-release /etc/system-release)
 if system_files.any? { |file| File.exist?(file) }
-  install_rules(iptables_rules)
+  success = install_rules(iptables_rules)
+  raise "#{0}: failed to install iptables rules" unless success
```

Comment:
Sorry. I think I had a moment of dyslexia. I read your comment backwards and looked at the wrong line. Fix submitted.